.storybook/config.js

```diff
@@ -61,8 +61,8 @@ const resetGlobals = () => {
 		isFirstAdmin: true,
```

Comment:
First thing's first – the changes here will likely be easier to review with whitespace changes hidden

![image](https://user-images.githubusercontent.com/1621608/105350696-35576580-5bf4-11eb-8326-a161d6eff0e7.png)